test/tsconfig.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": "../tsconfig",
   "compilerOptions": {
+    "strictNullChecks": false,
```

Comment:
```suggestion
```

Reply:
Both of these can be pretty easily resolved. I actually find strict null checks to actually be very helpful.